MantidQt/CustomInterfaces/src/Tomography/TomoToolConfigDialogCustom.cpp

```diff
@@ -0,0 +1,67 @@
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogCustom.h"
+#include "MantidQtCustomInterfaces/Tomography/ToolConfigCustom.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+const std::string TomoToolConfigDialogCustom::DEFAULT_TOOL_NAME =
+    "Custom command";
+
+// custom tool doesn't have a default method
+const std::string TomoToolConfigDialogCustom::DEFAULT_TOOL_METHOD = "";
+
+std::string TomoToolConfigDialogCustom::m_backupCommandLine = "";
+
+void TomoToolConfigDialogCustom::setupToolSettingsFromPaths() {
+
+  if (m_isInitialised) {
+    // None of the other paths matter, because the user could've changed
+    // them, so ignore them and load the current ones on the dialogue
+    QString run = m_customUi.lineEdit_runnable->text(); // current path
+    QString opts =
+        m_customUi.textEdit_cl_opts->toPlainText(); // current commands
```

Comment:
Could you do run and opts as follows:
`const std::string run (m_customUi.lineEdit_runnable->text().toStdString());`